tools/java/org/unicode/cldr/test/XPathToMenu.java

```diff
@@ -1,37 +1,40 @@
-package org.unicode.cldr.util;
+package org.unicode.cldr.test;
 
 import java.util.HashMap;
 import java.util.Map;
 
 import org.unicode.cldr.icu.LDMLConstants;
+import org.unicode.cldr.util.XPathParts;
 
 /**
- * @deprecated
+ * Deprecated utility class only used by ConsoleCheckCLDR.
+ *
+ * @deprecated Do not use.
  */
 @Deprecated
-public class PathUtilities {
-
+final class XPathToMenu {
     // ===== types of data and menu names
-    public static final String LOCALEDISPLAYNAMES = "//ldml/localeDisplayNames/";
+    private static final String LOCALEDISPLAYNAMES = "//ldml/localeDisplayNames/";
+    private static final String OTHER_CALENDARS_XPATH = "//ldml/dates/calendars/calendar";
+
+    private static final String CURRENCIES = "currencies";
+    private static final String TIMEZONES = "timezones";
+    private static final String METAZONES = "metazones";
+    private static final String MISC = "misc";
+    private static final String CODEPATTERNS = "codePatterns";
+    private static final String MEASNAMES = "measurementSystemNames";
+
+    private static final String LOCALEDISPLAYPATTERN_XPATH =
+            LOCALEDISPLAYNAMES + LDMLConstants.LOCALEDISPLAYPATTERN;
+    private static final String NUMBERSCURRENCIES =
+            LDMLConstants.NUMBERS + "/" + CURRENCIES;
+
     /**
      * All of the data items under LOCALEDISPLAYNAMES (menu items)
      */
-    public static final String LOCALEDISPLAYNAMES_ITEMS[] = { LDMLConstants.LANGUAGES,
+    private static final String[] LOCALEDISPLAYNAMES_ITEMS = { LDMLConstants.LANGUAGES,
         LDMLConstants.SCRIPTS, LDMLConstants.TERRITORIES, LDMLConstants.VARIANTS, LDMLConstants.KEYS,
-        LDMLConstants.TYPES, PathUtilities.CURRENCIES, PathUtilities.TIMEZONES,
```

Comment:
Ordering is very very odd here. The array is being defined before the constants in it are defined.
Normally IntelliJ would warn about this, but it wasn't. Maybe array initialization is different.
I reordered the constants to ensure everything is defined top-to-bottom.